export/main.ts

```diff
@@ -135,10 +135,12 @@ for(const proj of projects) {
  * Parse the label out of the job ID.
  */
 function parseLabel(data) {
-    let job_parts = data["job_link"].split('label=');
+    if(data.job_link) {
+        let job_parts = data["job_link"].split('label=');
```

Comment:
It's not related to this PR, but please use a URL query string parser. This will break if we add more query parameters like `&job=`.

Example: `new URL(data["job_link"]).searchParams.get("label")`

Reply:
This is really good feedback. I'll update to use the query string parser when I have a sec; just needed to get something out the door asap!